Makefile

```diff
@@ -11,4 +11,4 @@ setup:
 	@pip install -e .[tests] --quiet
 
 test: setup
-	@pyvows -c -l tc_aws
+	AWS_ACCESS_KEY_ID=test-key pyvows -c -l tc_aws
```

Comment:
@Bladrak  i think this might not be necessary when the `@mock_s3` decorator is in the right spot. it looked like it did not get called in some cases.